nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -133,6 +133,7 @@ let
         RestartSec = 20;
         EnvironmentFile = [ "/var/lib/mastodon/.secrets_env" ] ++ cfg.extraEnvFiles;
         WorkingDirectory = cfg.package;
+        LimitNOFILE = "1024000";
```

Comment:
apologies,I have no context of mastadon, but this value is a bit strange. is it 1024K files ?

Reply:
See systemd.exec(5). Thats the Number of File Descriptors

Reply:
I was just curious why that number, it looks like its just some arbitrary large number